human_experiments/scripts/run_experiment

```diff
@@ -251,7 +252,14 @@ pushd "$SCRIPTS_DIR" >/dev/null || exit 1
 
   if [[ $GOTO -le 18 ]]; then
     echo " "
-    text="[18] Type [${EMPH}quit${NC}] to end the experiment and close "
+    if yes_no_question "[18] Do you want to ${EMPH}rename images as timestamps${NC}?"; then
+      source rename_image_timestamp "$EXPERIMENT_DIR"
```

Comment:
No need to pass `$EXPERIMENT_DIR`. This variable is exported in the beginning of the uber script and therefore is visible by all subprocesses.

Reply:
Approving this since you can do it later, and it doesn't affect the code.